openapi/apis/core.json

```diff
@@ -2143,7 +2145,17 @@
           }
         },
         "operationId": "get-v1-nft-summary",
-        "description": "This endpoint allows you to retrieve the total NFT count for a given token address across multiple chains."
+        "description": "This endpoint allows you to retrieve the total NFT count for a given wallet address across multiple chains.",
+        "parameters": [
+          {
+            "schema": {
+              "type": "string"
+            },
+            "in": "query",
+            "name": "chain",
+            "description": "Chain name (example: bsc, ethereum, matic...)"
```

Comment:
list of chains for which count of NFTs required.

Reply:
ex. bsc,matic,ethereum

Reply:
Done